admin/tests/test_securedrop-admin-setup.py

```diff
@@ -119,3 +119,14 @@ def test_venv_cleanup_subprocess_exception(self, tmpdir, caplog):
                                                                       ':o')):
                 bootstrap.clean_up_tails3_venv(venv_path)
                 assert os.path.exists(venv_path)
+
+    def test_envsetup_cleanup(self, tmpdir, caplog):
+        venv_path = os.path.join(str(tmpdir), "empty_dir")
+        args = ""
+        with pytest.raises(subprocess.CalledProcessError):
+
+            with mock.patch('subprocess.check_output',
```

Comment:
OK so this diff looks good. one issue is that the test plan is not quite going to do the trick since at least for me `securedrop-admin setup` never times out (on Tails 4 beta2) so I'd need to ctrl-c which means the two cleanup lines won't execute. I have reproduced the original scenario however while testing other functionality on Tails, and I did need to manually remove the `admin/.venv3` directory in order to get `securedrop-admin setup` to complete on a subsequent call, so I think we should merge this.

Given that it's hard to test this scenario in Tails, we should ensure that the cleanup lines are being executed in this test by modifying this unit test to - as a side effect of the mocked `subprocess.check_output` call - create `venv_path` on disk. Then this test will verify that the cleanup works as expected.